src/Resources/Reporter.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace TestMonitor\Mantis\Resources;
+
+class Reporter extends Resource
```

Comment:
Could you rename this to User? Maybe you also want to use the assignee somewhere and it are all user objects.

Reply:
In api Reporter name is used, I can rename it to User but looks differ to api namming